src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -6349,6 +6351,122 @@ components:
         exclusive:
           $ref: '#/components/schemas/ShortIdProperty'
 
+    ##############################################################################
+    ## SNAPSHOT BUILDER API STANDARD MODELS
+    ##############################################################################
+    SnapshotBuilderConfig:
+      type: object
```

Comment:
I think all these objects need a `required:` clause. For example
```
      required:
        - domainOptions
        - programDataOptions
        - featureValueGroups
        - prepackagedConceptSets
```

Not all fields are required, such as `max` below.